adobe_vipm/flows/helpers.py

```diff
@@ -159,29 +159,19 @@ def __init__(self, is_validation=False):
 
     def __call__(self, client, context, next_step):
 
-        # Check if the Adobe customer has a 3YC commitment
-        commitment = get_3yc_commitment(context.adobe_customer)
+        # Get the 3YC commitment if it is enabled
+        commitment = self.get_3yc_commitment_enabled(context.adobe_customer)
 
-        if (
-            commitment
-            and commitment["status"] in (STATUS_3YC_COMMITTED, STATUS_3YC_ACTIVE)
-            and date.today() <= date.fromisoformat(commitment["endDate"])
-            and context.downsize_lines
-        ):
+        if commitment and context.downsize_lines:
```

Comment:
Please, can you put more details on this change - you remove 2 from 4 if conditions

Reply:
I have created a new method "get_3yc_commitment_enabled" to return the commitment only if it is enabled. These two conditions have been moved to a new method to reduce the method's complexity